.rubocop.yml

```diff
@@ -0,0 +1,124 @@
+AllCops:
```

Comment:
`inherit_from` is sufficient. We don't need a gem if we can download the config from github.